reference/resources/felixconfig.md

```diff
@@ -92,6 +92,11 @@ spec:
 | vxlanMTU                           | MTU to use for the VXLAN tunnel device. Also controls NodePort MTU when eBPF enabled. | int | int | `1410` |
 | vxlanPort                          | Port to use for VXLAN traffic. A value of `0` means "use the kernel default". | int | int | `4789` |
 | vxlanVNI                           | Virtual network ID to use for VXLAN traffic. A value of `0` means "use the kernel default". | int | int | `4096` |
+| wireguardEnabled                   | Enable encryption on WireGuard supported nodes in cluster. When enabled, pod to pod traffic will be sent over encrypted tunnels between the nodes. | `true`, `false` | boolean | `false` |
+| wireguardListeningPort             | Port used by WireGuard tunnels. Felix sets up WireGuard tunnel on each node specified by this port. Please note that this can only be setup in global `FelixConfiguration` resource. Setting it per host, config-file or environment variable will not work. | int | int | 51820 |
+| wireguardRoutingRulePriority       | WireGuard routing rule priority value set up by Felix. Change this to appropriate value if the default value can have an impact on existing routing rules on the node. | int | int | 99 |
```

Comment:
This sentence is not clear...are you saying user may need to change the value if the default doesn't work, or is this a warning that changing the default will impact existing routing rules on the node? 

WireGuard routing rule priority value set up by Felix. Change this to appropriate value if the default value can have an impact on existing routing rules on the node.